src/components/ApplicationForm/validate.ts

```diff
@@ -0,0 +1,22 @@
+const ERROR_EMPTY = 'Некорректно заполненное поле';
+//TODO можно добавить свитч кейс для разных правил валидации (имейл и тд)
```

Comment:
TODO комментарии корректно пишутся так:
```suggestion
// TODO: Можно добавить свитч кейс для разных правил валидации (имейл и тд)
```